pytest_harvest/plugin.py

```diff
@@ -498,3 +498,14 @@ def possibly_restore_xdist_workers_structs(session):
                 else:
                     assert len(set(saved_fixture_dct.keys()).intersection(set(_saved_fixture_dct.keys()))) == 0
                     saved_fixture_dct.update(_saved_fixture_dct)
+
+
+def doctestable():
+    """Do nothing, but have a doctest.
+
+    Examples
+    --------
+    >>> 1 + 1
+    2
+    """
+    return
```

Comment:
Cant we have this doctest somewhere in the `tests/` folder rather ? I'm a bit reluctant to leave it in the "main" part.

Reply:
I don't know if `--doctests` works for things in `tests/`, but I'll give it a shot

Reply:
ok. If it ends up being overly complex to move this to tests/, another option is simply to leave it here, but to add an underscore to the name to make it private (if doctest is happy with it).